draft-hendrickson-privacypass-public-metadata-issuance.md

```diff
@@ -56,7 +278,20 @@ TODO Security
 
 # IANA Considerations
 
-This document has no IANA actions.
+This extends the token type registry defined in {{Section 8.2.1 of !PROTOCOL}} with a new token type of value `0xDA7A`:
+
+* Value: 0xDA7A
+* Name: Partially Blind RSA (4096-bit)
+* Token Structure: As defined in {{public-request}}
+* TokenChallenge Structure: As defined in {{Section 2.1 of AUTHSCHEME}}
+* Publicly Verifiable: Y
+* Public Metadata: Y
+* Private Metadata: N
+* Nk: 256
```

Comment:
This suggests the key size is 2048 bits, not 4096. @smhendrickson can we make sure this is consistent throughtout?

Reply:
Done in https://github.com/smhendrickson/draft-hendrickson-privacypass-public-metadata-issuance/pull/1/commits/f3b1e72f8794d0e92172a148a016b557bbb555ff